source/glue/normalizers/phone_normalizer.py

```diff
@@ -0,0 +1,39 @@
+import phonenumbers
+
+restrictedDigits = {
```

Comment:
Could you leave a comment on why restrictedDigits is needed?

Reply:
added here: [37df466](https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/72/commits/37df466888ed8508ceafd9a36c438887d459be6e)